src/components/stepper/CircleStepIcon.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/styles'
+import Theme from '../../theme-manager/themes/model'
+import FontAwesomeIcon from '../FontAwesomeIcon'
+import Text from '../../elements/Text'
+import clsx from 'clsx'
+
+interface StepIconProps {
+  stepNumber: number
+  icon?: string
+  inactiveStep?: boolean
+  className?: string
+  color?: string
+}
+
+export default function CircleStepIcon({
+  stepNumber,
+  icon,
+  className,
+  color,
+  inactiveStep = false,
+}: StepIconProps) {
+  const classes = useStyles({ color, inactiveStep })
+  return (
+    <div className={clsx(classes.circle, className)}>
+      {icon ? (
+        <FontAwesomeIcon className={clsx(classes.icon, 'icon')}>{icon}</FontAwesomeIcon>
+      ) : (
+        <Text variant="body1" className={clsx(classes.stepNumber, 'step-number')}>
```

Comment:
do we need the step-number class for anything? I don't see it referenced anywhere (same for the 'icon' class and several others I see in the other file as well). I could see them being useful to reference in a parent component, but just checking

Reply:
I just gave each element a name so that it can be easily referenced and customized by a parent component 

Like this

```
const App = () => {
   const classes = useStyles()
   return <CircleStepIcon stepNumber={1} className={classes.stepIcon} />
}

const useStyles = makeStyles(() => ({
  stepIcon: {
    backgroundColor: 'red',
    '& .step-number': {
      fontStyle: 'italic',
      fontWeight: 600
    },
  },
}))
```